packages/tocco-ui/src/FormattedValue/typeFormatters/DurationFormatter.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {FormattedDate} from 'react-intl'
+
+const DurationFormatter = props => {
+  const milliSeconds = parseInt(props.value || 0)
```

Comment:
parseInt("asdf" || 0) returns NaN. I think it should be parseInt(props.value) || 0